poi-ooxml/src/test/java/org/apache/poi/xdgf/usermodel/section/geometry/TestEllipticalArcTo.java

```diff
@@ -0,0 +1,76 @@
+package org.apache.poi.xdgf.usermodel.section.geometry;
```

Comment:
could you add license headers to the new files?

Reply:
Just pushed the changes. Headers are added to this file and `TestGeometryUtils.java`.